package.json

```diff
@@ -22,10 +21,12 @@
     "format:yaml": "prettier --write --parser yaml '**/*.{yaml, yml}'",
     "format:graphql": "prettier --write --parser graphql '**/*.graphql'",
     "start": "yarn playground:start",
-    "prerelease": "yarn auth && yarn build && yarn typecheck",
```

Comment:
Not sure why we run `yarn typecheck` during a release, it should not be necessary.